webapp/channels/src/components/onboarding_tasklist/onboarding_tasklist.tsx

```diff
@@ -332,7 +334,7 @@ const OnBoardingTaskList = (): JSX.Element | null => {
                             <Skeleton>
                                 <img
                                     src={checklistImg}
-                                    alt={'On Boarding video'}
```

Comment:
Yeah, I'm trying not to get too distracted by them at the moment, but we could definitely use a pass over our existing aria-labels and alt texts since there's a lot of ones that could be improved. Strictly speaking, we meet the requirements, but they're not all great